totpauth-impl/src/main/resources/views/totp-register.vm

```diff
@@ -58,6 +59,13 @@ p.error {
             </div>
           </form>
           <li><a href="https://support.google.com/accounts/answer/1066447" target="_blank">Google Authenticator installation instructions</a></li>
+<li><a href="https://support.authy.com/hc/en-us/articles/360006303934-Add-a-New-Two-Factor-Authentication-2FA-Account-Token-in-the-Authy-App" target="_blank">Authy Authenticator installation instructions</a></li>
+<li><a href="https://support.microsoft.com/en-us/account-billing/set-up-an-authenticator-app-as-a-two-step-verification-method-2db39828-15e1-4614-b825-6e2b524e7c95" target="_blank">Microsoft Authenticator installation instructions</a></li>
+<li><a href="https://support.lastpass.com/s/document-item?language=en_US&bundleId=lastpass&topicId=LastPass/enable-mfa.html&_LANG=enus" target="_blank">LastPass Authenticator installation instructions</a></li>
```

Comment:
Reading this link, it refers to the process of adding a 2nd factor for authentication to your LastPass account. It's not about using LastPass to manage 2-factor authentication to other accounts. Is that definitely supported by LastPass?

Reply:
I think this is the more relevant link: https://support.lastpass.com/s/document-item?language=en_US&bundleId=lastpass&topicId=LastPass/create-totp-vault.html&_LANG=enus

Reply:
I agree, change made.